src/extractor/scripting_environment_lua.cpp

```diff
@@ -34,7 +34,12 @@ namespace
 template <class T>
 auto get_value_by_key(T const &object, const char *key) -> decltype(object.get_value_by_key(key))
 {
-    return object.get_value_by_key(key, "");
+    auto v = object.get_value_by_key(key);
```

Comment:
clang-format-3.8 -style=file -i path/to/file.cpp

We can do this for you before merging, if you don't have the tooling for it installed.